readme.md

```diff
@@ -0,0 +1,26 @@
+# Laravel Media plugin
+
+## Introduction
+
+This plugin adds the ability for users to work with media files.
+
+## Installation
+
+1. Install the package using the following command: `composer require ronasit/laravel-media`
+2. Run `php artisan vendor:publish`
+3. Add `src/MediaServiceProvider` to `config/app.php` your application
```

Comment:
this step requires for the Laravel < 5.5

Reply:
Fixed.